ReadingList.Domain/Services/Sql/Implementations/UserSqlService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Cinch.SqlBuilder;
+using ReadingList.Domain.Services.Sql.Interfaces;
+
+namespace ReadingList.Domain.Services.Sql
+{
+    public class UserSqlService : IUserSqlService
+    {
+        private readonly ISqlBuilder _sqlBuilder;
+
+        public UserSqlService()
+        {
+            _sqlBuilder = new SqlBuilder()
+                .Select("Id", "Login", "Password", "ProfileId", "(SELECT Name FROM Roles WHERE Id = RoleId) AS Role")
+                .From("Users");
+        }
+
+        public string GetUserByLoginSql(string parameterName = "login") =>
```

Comment:
not sure that `parameterName` is a right name for argument  
can be named as login
and what was the intention to defined default value `login`?